client/src/reducers/homepageReducer.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * This file exports homepage reducer that will handle all dispatched client-homepage-related actions.
+ */
+import { FETCH_HOMEPAGE, UPDATE_HOMEPAGE } from '../actions/types';
+
+const INITIAL_HOMEPAGE_STATE = {
+  aboutUs: [''],
```

Comment:
I was also curious about @pmyip3 comment on last PR. Does this have to a list of strings? why not just a plain string?

Reply:
Okay I read your comment. IMO this is fine but it's not very self explanatory why we're resorting to using a string list. Can you add docs here (and perhaps in the API) as well to explain why we're using a string list? This should make it more clearer

Reply:
Ok, will add comment here and other files as well.